scripts/linters/pylint_extensions.py

```diff
@@ -411,19 +407,17 @@ class DocstringParameterChecker(checkers.BaseChecker):
             'Missing yield type documentation',
             'missing-yield-type-doc',
             'Please document the type yielded by this method.',
-            # We can't use the same old_name for two different warnings
-            # {'old_names': [('W9009', 'missing-yields-doc')]}.
         ),
         'W9015': (
             '"%s" missing in parameter documentation',
             'missing-param-doc',
             'Please add parameter declarations for all parameters.',
-            {'old_names': [('W9003', 'missing-param-doc')]}),
+            {'old_names': [('W9003', 'old-missing-param-doc')]}),
         'W9016': (
             '"%s" missing in parameter type documentation',
             'missing-type-doc',
-            'Please add parameter type declarations for all parameters.',
-            {'old_names': [('W9004', 'missing-type-doc')]}),
```

Comment:
I think this is similar to dealing with the above case with `W9015` and `W9003` - for both the places we were using old_names same as the warning name. A single decision should be taken for both - remove both `old_names` or rename both `old_names`.

Reply:
Yeah, probably up to you and @DubeySandeep. I'm fine with anything that you suggest.